src/subdomains/generic/kyc/dto/mapper/kyc-info.mapper.ts

```diff
@@ -43,7 +43,7 @@ export class KycInfoMapper {
 
   // --- HELPER METHODS --- //
   private static getUiSteps(userData: UserData): KycStep[] {
-    if (userData.isKycTerminated) return [];
+    if (userData.isBlocked || userData.isDeleted) return [];
```

Comment:
I think we can revert this. Only if the KYC is terminated we want to hide the KYC steps